modules

```diff
@@ -0,0 +1,5 @@
+GLUON_SITE_FEEDS='ffmd'
+
+PACKAGES_FFMD_REPO=https://github.com/FreifunkMD/ffmd-packages
```

Comment:
This should be a valid Git repository URL? What you copy from GitHub to clone is https://github.com/FreifunkMD/ffmd-packages.git

Reply:
They both work and I definitely test-built it (used for package development).

Github directly resolves the *.git-URLs to the format I gave. What is the rationale in adding .git?